utils/src/server/rpc_module.rs

```diff
@@ -780,16 +759,15 @@ mod tests {
 		let (sub_id, mut my_sub_stream) = module.test_subscription("my_sub", None).await;
 		for i in (0..=2).rev() {
 			let my_sub = my_sub_stream.next().await.unwrap();
-			let my_sub = serde_json::from_str::<JsonRpcNotification<JsonRpcSubscriptionParams<char>>>(&my_sub).unwrap();
+			let my_sub = serde_json::from_str::<Notification<SubscriptionPayload<char>>>(&my_sub).unwrap();
```

Comment:
can't you use the `SubscriptionResponse` type alias here instead?

Reply:
Ofc I can, that's the very reason I added `SubscriptionResponse`! I messed up a merge I think. Ty!